backend/tournesol/views/video.py

```diff
@@ -154,13 +159,24 @@ def get_queryset(self):
             for crit in settings.CRITERIAS
         ]
         criteria_weight = Case(*criteria_cases, default=0)
-        queryset = (
-            queryset.annotate(
-                total_score=Sum(F("criteria_scores__score") * criteria_weight)
-            )
-            .filter(total_score__gt=0)
-            .order_by("-total_score")
+
+        unsafe = request.query_params.get('unsafe')
+        show_unsafe = False
+        if unsafe:
+            show_unsafe = unsafe == 'true'
+
+        queryset = queryset.annotate(
+            total_score=Sum(F("criteria_scores__score") * criteria_weight)
         )
+
+        if show_unsafe is True:
+            queryset = queryset.order_by("-total_score")
+        else:
+            queryset = (
+                queryset
+                .filter(total_score__gt=0)
+                .order_by("-total_score")
+            )
```

Comment:
When `show_unsafe` is True, we would also expect to include videos without a minimum of contributions. So the filter applied on line 112 needs to be moved too.

Reply:
I miss it thanks

Reply:
I would suggest to move the filter on `rating_n_contributors` here in the same condition. I think that it would help the reader to grasp all what is going on with this safe/unsafe filters.

Reply:
Thanks I updated